### ISSUE TITLE
Let it possible to omit 1 from association end multiplicity

### ISSUE THREAD

**Issue by nboldi** [ELTE-Soft/txtUML#30]
Currently an association can be declared like this:
```
	association AB {
		public 1 A a;
		public * B b;
	}
```

It would be much more nicer to write it implicitely declaring `a` as single multiplicity:
```
	association AB {
		public A a;
		public *B b;
	}
```

Omitting 1 is not only forbidden, but produces an exception:
```
java.lang.NullPointerException
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer.calculateApiSuperType(XtxtUMLJvmModelInferrer.java:519)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer.lambda$14(XtxtUMLJvmModelInferrer.java:452)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer$$Lambda$26/1656456005.apply(Unknown Source)
	at org.eclipse.xtext.xbase.jvmmodel.JvmTypesBuilder.initializeSafely(JvmTypesBuilder.java:206)
	at org.eclipse.xtext.xbase.jvmmodel.JvmTypesBuilder.toClass(JvmTypesBuilder.java:384)
	at org.eclipse.xtext.xbase.jvmmodel.JvmTypesBuilder.toClass(JvmTypesBuilder.java:361)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer._toJvmMember(XtxtUMLJvmModelInferrer.java:455)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer.toJvmMember(XtxtUMLJvmModelInferrer.java:670)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer.lambda$5(XtxtUMLJvmModelInferrer.java:290)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer$$Lambda$25/500951252.apply(Unknown Source)
	at org.eclipse.xtext.xbase.jvmmodel.JvmTypesBuilder.initializeSafely(JvmTypesBuilder.java:206)
	at org.eclipse.xtext.xbase.jvmmodel.JvmTypesBuilder.toClass(JvmTypesBuilder.java:384)
	at org.eclipse.xtext.xbase.jvmmodel.JvmTypesBuilder.toClass(JvmTypesBuilder.java:361)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer._toJvmMember(XtxtUMLJvmModelInferrer.java:294)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer.toJvmMember(XtxtUMLJvmModelInferrer.java:654)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer.lambda$0(XtxtUMLJvmModelInferrer.java:93)
	at hu.elte.txtuml.xtxtuml.jvmmodel.XtxtUMLJvmModelInferrer$$Lambda$1/1059378513.apply(Unknown Source)
	at org.eclipse.xtext.xbase.jvmmodel.JvmModelAssociator$1.run(JvmModelAssociator.java:397)
	at org.eclipse.xtext.xbase.jvmmodel.JvmModelAssociator.installDerivedState(JvmModelAssociator.java:407)
	at org.eclipse.xtext.resource.DerivedStateAwareResource.installDerivedState(DerivedStateAwareResource.java:240)
	at org.eclipse.xtext.xbase.resource.BatchLinkableResource.getContents(BatchLinkableResource.java:148)
	at org.eclipse.xtext.xbase.typesystem.internal.LogicalContainerAwareBatchTypeResolver.getEntryPoints(LogicalContainerAwareBatchTypeResolver.java:44)
	at org.eclipse.xtext.xbase.typesystem.internal.DefaultBatchTypeResolver.getTypeResolver(DefaultBatchTypeResolver.java:63)
	at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver$1.get(CachingBatchTypeResolver.java:49)
	at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver$1.get(CachingBatchTypeResolver.java:1)
	at org.eclipse.xtext.util.OnChangeEvictingCache.get(OnChangeEvictingCache.java:77)
	at org.eclipse.xtext.xbase.typesystem.internal.CachingBatchTypeResolver.doResolveTypes(CachingBatchTypeResolver.java:46)
	at org.eclipse.xtext.xbase.typesystem.internal.AbstractBatchTypeResolver.resolveTypes(AbstractBatchTypeResolver.java:69)
	at org.eclipse.xtext.xbase.resource.BatchLinkingService.resolveBatched(BatchLinkingService.java:71)
	at org.eclipse.xtext.xbase.resource.BatchLinkableResource.resolveLazyCrossReferences(BatchLinkableResource.java:165)
	at org.eclipse.xtext.EcoreUtil2.resolveLazyCrossReferences(EcoreUtil2.java:528)
	at org.eclipse.xtext.ui.editor.reconciler.XtextDocumentReconcileStrategy.postParse(XtextDocumentReconcileStrategy.java:175)
	at org.eclipse.xtext.ui.editor.reconciler.XtextDocumentReconcileStrategy.doReconcile(XtextDocumentReconcileStrategy.java:153)
	at org.eclipse.xtext.ui.editor.reconciler.XtextDocumentReconcileStrategy.reconcile(XtextDocumentReconcileStrategy.java:67)
	at org.eclipse.xtext.ui.editor.reconciler.XtextReconciler.doRun(XtextReconciler.java:442)
	at org.eclipse.xtext.ui.editor.reconciler.XtextReconciler.access$3(XtextReconciler.java:422)
	at org.eclipse.xtext.ui.editor.reconciler.XtextReconciler$1.process(XtextReconciler.java:363)
	at org.eclipse.xtext.ui.editor.reconciler.XtextReconciler$1.process(XtextReconciler.java:1)
	at org.eclipse.xtext.util.concurrent.IUnitOfWork$Void.exec(IUnitOfWork.java:37)
	at org.eclipse.xtext.resource.OutdatedStateManager.exec(OutdatedStateManager.java:121)
	at org.eclipse.xtext.ui.editor.model.XtextDocument$XtextDocumentLocker.modify(XtextDocument.java:428)
	at org.eclipse.xtext.ui.editor.model.XtextDocument.internalModify(XtextDocument.java:162)
	at org.eclipse.xtext.ui.editor.reconciler.XtextReconciler.run(XtextReconciler.java:360)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
```

**Comment by devaigergely81** [comment]
I agree that the exception needs to be eliminated, but I'm not sure about the syntax proposal. If the multiplicity is mandatory, the language is more regular, it is closer to JtxtUML, easier to process when exported. Let's discuss this with the teams!

**Comment by djnemeth** [comment]
This is implemented now, see #243.
